README.md

```diff
@@ -288,25 +288,21 @@ sysroot. The final sysroot, the stage 1 sysroot, will contain both the `std` and
 
 Xargo lets you create a sysroot with custom crates. You can virtually put any
 crate in the sysroot. However, this feature is mainly used to create [alternative
-`std` facades][steed], and to replace the `test` crate with [one that supports
+`std` facades][rust-3ds], and to replace the `test` crate with [one that supports
 `no_std` targets][utest]. To specify the contents of the sysroot simply list the
 dependencies in the Xargo.toml file as you would do with Cargo.toml:
 
-[steed]: https://github.com/japaric/steed
+[steed]: https://github.com/rust3ds/rust3ds-template
 [utest]: https://github.com/japaric/utest
 
 ``` toml
-[dependencies]
-collections = {}
-rand = {}
-
-[dependencies.compiler_builtins]
-features = ["mem"]
-stage = 1
+[dependencies.alloc]
+[dependencies.panic_abort]
+[dependencies.panic_unwind]
 
 [dependencies.std]
-git = "https://github.com/japaric/steed"
-stage = 2
+git = "https://github.com/rust3ds/ctru-rs"
+stage = 1
 ```
```

Comment:
One case that seems to come up regularly is something like
```
[patch.crates-io]
compiler_builtins = { ... }
```
Is that worth documenting here?

Reply:
You're talking about patch in general, not specifically for compiler-builtins right?

Yeah, it is probably worth documenting somewhere. It can be useful to patch a handful of dependencies without doing intrusive changes (like using a custom libc, custom compiler_builtins, etc...).

Reply:
Custom compiler_builtins seems like a common case based on the bugreports we get, but maybe that impression is wrong. Yeah, any example will do.